lib/src/components/NxPageHeader/__tests__/NxPageHeader.test.tsx

```diff
@@ -5,84 +5,119 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 import React from 'react';
-import { shallow } from 'enzyme';
-import 'jest-enzyme';
-
-import * as enzymeUtils from '../../../__testutils__/enzymeUtils';
 import NxPageHeader from '../NxPageHeader';
-import AbstractNxPageHeader from '../../AbstractNxPageHeader/AbstractNxPageHeader';
 
-describe('NxPageHeader', function() {
-  const getShallowComponent = enzymeUtils.getShallowComponent(NxPageHeader, {});
+import { screen } from '@testing-library/react';
+import { rtlRenderElement } from '../../../__testutils__/rtlUtils';
+
+fdescribe('NxPageHeader', function() {
+
+  const renderEl = rtlRenderElement(NxPageHeader, {});
 
-  it('renders an AbstractNxPageHeader', function() {
-    expect(getShallowComponent()).toMatchSelector(AbstractNxPageHeader);
+ it('renders a simple NxPageHeader', function() {
+    renderEl();
+    const logo = screen.getByRole('img')
+    expect(logo).toBeInTheDocument();
+    
   });
 
-  it('passes its links, homeLink, and children props to the AbstractNxPageHeader', function() {
-    const props = {
-          links: [{ name: 'foo', href: '#bar' }],
-          homeLink: '#home',
-          children: <button>Click Here For a Free iPhone</button>
-        },
-        component = getShallowComponent(props);
+  it('renders a custom logo when given logo prop', function() {
+    const logo = renderEl({ logo: { path: 'foo', alt: 'bar' } })?.querySelector("img");
```

Comment:
This should be written in terms of RTL queries and roles, not CSS selectors and `querySelector`

Reply:
> This should be written in terms of RTL queries and roles, not CSS selectors and `querySelector`

Fixed: [4ae11b1](https://github.com/sonatype/sonatype-react-shared-components/pull/935/commits/4ae11b1ffaacfcf331ad8f3252b2685d95844a84)